### ISSUE TITLE
Enable users to provide private feedback to Safety Team if they rate "bad experience" on slider during reference flow

### ISSUE THREAD

**Issue by jesseallhands** [Couchers-org/couchers#5181]
Note: This ticket is dependent on issue #4121 which must be completed first. 

### **Problem:**
When leaving a reference for someone, there is no way for a user to give private feedback to the safety team if they rate their experience "bad experience" on the slider page.

### **Solution:**
When leaving a reference, users should be given the opportunity to privately provide information to the Safety Team if they move the slider to rate the experience "bad experience". 

Screenshot 1 below shows an example of a "bad experience" rating. A "bad experience" is defined as a rating of <=0.4 on the slider scale of 0.0 to 1.0. This may seem high, but is on purpose. The vast majority of experiences are >0.4.  The experience rating page with the slider (part of the reference flow) needs to be updated to take the user to the "private feedback" page being built on issue #4121 (screenshot 2 and 3 below).

Here is the desired flow/details:

1. If the user moves the slider anywhere <=0.4, upon clicking the "next" button, they should be taken to a "private feedback" page (see issue #4121 ) where they can share what happened with the Safety Team.
   - At the top of the "private feedback" page, note that the text should read "You answered that you had a negative experience." instead of "You answered that you felt unsafe with this person’s behavior." 
3. If the user previously entered text on the "private feedback" page due to the previous "feel safe? No" step of the reference flow, that text must appear in the text box, and the text on the page must be updated (see differences in text between Screenshot 2 and Screenshot 3 below).
   - This step allows the user to update the feedback they provided during the previous step or add additional information
   - The updated text (instructions) keep the user from accidentally overwriting/deleting what they previously wrote.
4. Once the reference is submitted at the end of the reference flow, the private feedback will be sent to safety@couchers.org and stored on the backend using the existing "report" functionality.

Screenshot 1
Here is what the slider page looks like with a "bad experience" rating:
![image](https://github.com/user-attachments/assets/e6420858-1b27-4d9f-9dea-3a156d9d4ee4)

Screenshot 2
Here is what the "private feedback" page should look like after rating "bad experience", assuming the text box is empty (because the user did not previously submit feedback to the Safety Team). Note the various differences in texts compared to Screenshot 3:
![image](https://github.com/user-attachments/assets/23f5400c-4654-4812-8c7c-19b8d33c0887)

Screenshot 3
Here is what the "private feedback" should look like after rating "bad experience", and the text box CONTAINS TEXT because the user entered feedback after the previous "felt unsafe? no" step. Note the various differences in texts copared to Screenshot 2:
![image](https://github.com/user-attachments/assets/6ea1b9fe-97e5-4a3e-990c-513e124e0e16)

Components:

- [ X ] Backend changes to enable appending the submitted text to the DB
- [ X ] Backend changes to enable notification of support at safety@couchers.org (similar mechanic as reports currently work)
- [ ] Frontend: Update navigation of "next" button when slider rating is "bad experience" so user is taken to "private feedback" page.
- [ ] Ensure that texts on the "private feedback" page match Screenshot 2 and 3.
- [ ] UI/UX Testing

**Comment by bakeiro** [comment]
> Screenshot 1 below shows an example of a "bad experience" rating. A "bad experience" is defined as a rating of <=0.4

I think this is not correct, since the bad ratings are <0.33, because 0.33 = neutral

**Comment by jesseallhands** [comment]
> I think this is wrong, since the bad ratings are <0.33, because 0.33 = neutral

<0.33 would be better in my opinion. But aapeli was pretty certain that originally it was <=0.4 which is why I wrote it in this issue. If it really is <=0.33 that's perfect! Less "false positives" for the support team (and less bugging users to give private feedback when it's not necessary).

**Comment by nabramow** [comment]
@jesseallhands Do you remember what this is blocked by?

**Comment by jesseallhands** [comment]
@nabramow it's dependent on #4121 but it's basically the same task

**Comment by nabramow** [comment]
Closing in favor of #5508
